dll/win32/msgina/gui.c

```diff
@@ -191,7 +190,10 @@ StatusDialogProc(
 
             if (DlgData_LoadBitmaps(pDlgData))
             {
-                pDlgData->TimerID = SetTimer(hwndDlg, -1, 20, NULL);
+                if (SetTimer(hwndDlg, IDT_BAR, 20, NULL) == 0)
+                {
+                    ERR("SetTimer(IDT_BAR) failed: %d\n", GetLastError());
```

Comment:
Not sure whether a thing such as the failure to create a timer to make cute animation bar rolling around, is worth an error in the debug log, especially knowing that if such function fails it's because your system is completely out of resources....

Reply:
I concur this is debug log only and unlikely to happen. (And harmless, visually noticeable, ...)
Nonetheless, I think it is a good pattern to use: I let reviewer decide.